diff --git a/libobs/CMakeLists.txt b/libobs/CMakeLists.txt
index d2e26710d..cdc82fd6a 100644
--- a/libobs/CMakeLists.txt
+++ b/libobs/CMakeLists.txt
@@ -291,7 +291,17 @@ set(public_headers
     util/profiler.h
     util/sse-intrin.h
     util/text-lookup.h
-    util/util_uint64.h)
+    util/util_uint64.h
+    # begin patch for obs-ndi
+    util/util_uint64.h
+    media-io/audio-resampler.h
+    media-io/video-frame.h
+    util/circlebuf.h
+    util/config-file.h
+    util/platform.h
+    util/threading.h
+    util/threading-posix.h)
+    # end patch for obs-ndi    
 
 if(ENABLE_HEVC)
   list(APPEND public_headers obs-hevc.h)
